src/dat/sys/figwheel_server.clj

```diff
@@ -0,0 +1,29 @@
+(ns dat.sys.figwheel-server
```

Comment:
Yes actually. That gives us a namespace for start.cljs as well. Makes everything cleaner and I can move figwheel_server.clj to the dev folder.